contracts/TreasuryActions.cdc

```diff
@@ -293,4 +293,82 @@ pub contract TreasuryActionsV4 {
       self.intent = "Update the threshold of signers to ".concat(threshold.toString()).concat(".")
     }
   }
-}
\ No newline at end of file
+
+  pub struct AddSignerUpdateThreshold: MyMultiSigV4.Action {
+    pub let threshold: UInt
+    pub let signer: Address
+    pub let intent: String
+    pub let proposer: Address
+
+    access(account) fun execute(_ params: {String: AnyStruct}) {
+      let treasuryRef: &DAOTreasuryV4.Treasury = params["treasury"]! as! &DAOTreasuryV4.Treasury
+      let manager = treasuryRef.borrowManager()
+
+      // Add Signer
+      manager.addSigner(signer: self.signer)
```

Comment:
What do you say about creating a one manager function to do both?

Reply:
I was thinking about this because we could consolidate the 2 actions AddSignerUpdateThreshold and RemoveSignerUpdateThreshold into one.

It would DRY the code a little bit, but it would be a small optimization, and I like the clarity of having two explicitly separate actions.